### DISCUSSION TITLE
commande pour voir les gens en prison

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#1803] [Nouvelles idées]
on fait une commande et paf les mec en prinson s affichent comme pour le top et comme sa c est plus facile pour la campagne si on est sans amis

**Comment by BastLast** [comment]
Sinon tu invite un de tes amis a jouer à draftbot et tu attend qu'il finisse en prison 😊

**Reply by kekeishere** [reply]
sa peut prendre disons... beaucoup de temps :)

**Reply by BastLast** [reply]
Oue mais cette mission n'apparaît qu'une fois pr le moment

**Comment by Akutana** [comment]
Ça enlève tout l'intérêt de la mission :pQuack:
